### PR TITLE
added export and save buttons below preview

### LINKED ISSUE #463: add "save" and "export" buttons at the end of the page (after preview)
use the same logic as in the menubar

### PR DESCRIPTION
**Pull request by Epsilon02**
close #463

### CODE REVIEW

File: src/app/menubar/menubar.component.html [hunk 1]
```diff
@@ -72,5 +72,23 @@
     <div #preview>
       <app-preview></app-preview>
     </div>
+    <div class="buttons">
+      <button
+        class="unselectable"
+        mat-raised-button
+        color="primary"
+        (click)="exportFile({ name: 'CaBr2', extensions: ['cb2'] })"
```
Comment (crapStone) [inline src/app/menubar/menubar.component.html:80]: duplicated code!

should we create an extra function or a mapping from which these object can be pulled?

**Review by crapStone — CHANGES_REQUESTED** [review]

File: src/app/menubar/menubar.component.html [hunk 2]
```diff
@@ -6,10 +6,10 @@
     <mat-menu #menu="matMenu">
       <button mat-menu-item (click)="newDocument()">{{ strings.menu.newDocument }}</button>
       <button mat-menu-item (click)="loadFile()">{{ strings.menu.loadFile }}</button>
-      <button mat-menu-item (click)="exportFile({ name: 'CaBr2', extensions: ['cb2'] })">
+      <button mat-menu-item (click)="exportCab2File()">
```
Comment (crapStone) [inline src/app/menubar/menubar.component.html:9]:
```suggestion
      <button mat-menu-item (click)="exportCB2File()">
```

**Review by crapStone — CHANGES_REQUESTED** [review]

File: src/app/menubar/menubar.component.ts [hunk 3]
```diff
@@ -259,4 +232,39 @@ export class MenubarComponent implements OnInit {
       this.darkModeSwitched.emit(checked);
     });
   }
+
+  exportCab2File(): void {
```
Comment (6543) [inline src/app/menubar/menubar.component.ts:236]:
```suggestion
  exportCB2File(): void {
```

File: src/app/menubar/menubar.component.html [hunk 4]
```diff
@@ -72,5 +72,23 @@
     <div #preview>
       <app-preview></app-preview>
     </div>
+    <div class="buttons">
+      <button
+        class="unselectable"
+        mat-raised-button
+        color="primary"
+        (click)="exportCab2File()"
```
Comment (6543) [inline src/app/menubar/menubar.component.html:80]:
```suggestion
        (click)="exportCB2File()"
```

**Comment by 6543** [comment]
this will ad those buttons, but can we put them to at the center or to the right side?

**Comment by Epsilon02** [comment]
I'm not sure if it make sense. I think the user have the cursor more on the left side because the most user interactions are on the left side to. Correct me if I'm wrong.

**Review by 6543 — APPROVED** [review]

**Pull request merged**